<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHM</class>
 <widget class="QMainWindow" name="IHM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>648</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>IHM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutTransf">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxTransf">
       <property name="title">
        <string>Transform</string>
       </property>
       <widget class="QRadioButton" name="radioButtonDCT">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>DCT</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonIDCT">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>IDCT</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutSignalWindow">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxSignalWindow">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Signal Window</string>
       </property>
       <widget class="QTimeEdit" name="timeEditStart">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>50</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MinuteSection</enum>
        </property>
        <property name="displayFormat">
         <string>mm:ss:zzz</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="timeEditStop">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>90</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="displayFormat">
         <string>mm:ss:zzz</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStartSignalWindow">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Start :</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStopSignalWindow">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>(Format : M:s:ms)</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="enableSignalWindow">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonWindowing">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>179</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Windowing</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutAction">
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClear">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFocus">
       <property name="text">
        <string>Focus</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="graphCurve" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>691</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graphTransf" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>691</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>0</y>
      <width>151</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutInput">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxInput">
       <property name="title">
        <string>Input</string>
       </property>
       <widget class="QLabel" name="labelIn">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>15</y>
          <width>131</width>
          <height>201</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>250</y>
      <width>151</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutOuput">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxOutput">
       <property name="title">
        <string>Output</string>
       </property>
       <widget class="QLabel" name="labelOut">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>201</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>701</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutCalcWindow">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxCalcWindow">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Calculation Window</string>
       </property>
       <widget class="QLabel" name="labelCalcWindow">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Size :</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="enableCalcWindow">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxCalcWindow">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelCalcWindow_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>(nb of ech)</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxCalcWindow">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelCalcWindow_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Type :</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpMe"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionWave">
   <property name="text">
    <string>Wave</string>
   </property>
  </action>
  <action name="actionTexte">
   <property name="text">
    <string>Texte</string>
   </property>
  </action>
  <action name="actionHelpMe">
   <property name="text">
    <string>Help me</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCSV">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
